src/syscall/syscall_windows.go

```diff
@@ -360,6 +360,11 @@ func Open(path string, mode int, perm uint32) (fd Handle, err error) {
 			}
 		}
 	}
+
+	if mode&O_SYNC != 0 {
+		attrsAndFlags != FILE_FLAG_WRITE_THROUGH
```

Comment:
I don't believe this line actually does anything, just checks if those two values are not equal, returning a bool that isn't being captured

Reply:
Thank you @BrianLeishman. It was a silly typo. I've fixed it :+1: